.github/CONTRIBUTING.rst

```diff
@@ -12,31 +12,29 @@ Build Status
 
   + `Stable buildbots <http://buildbot.python.org/3.6.stable/>`_
 
-- 3.5
-
-  + `Stable buildbots <http://buildbot.python.org/3.5.stable/>`_
-
 - 2.7
 
   + `Stable buildbots <http://buildbot.python.org/2.7.stable/>`_
 
 
 Contribution Guidelines
 -----------------------
-Please read the `devguide <https://cpython-devguide.readthedocs.io/>`_ for
+Please read the `devguide <https://devguide.python.org/>`_ for
 guidance on how to contribute to this project. The documentation covers
 everything from how to build the code to submitting a pull request. There are
 also suggestions on how you can most effectively help the project.
 
 Please be aware that our workflow does deviate slightly from the typical GitHub
 project. Details on how to properly submit a pull request are covered in
-`Lifecycle of a Pull Request <https://cpython-devguide.readthedocs.io/pullrequest.html>`_.
-One key point is to keep comments on GitHub to only those related to the reviewing
-the code in the pull request. All other discussions -- e.g. about the issue being
-fixed -- should happen on bugs.python.org.
-
-If you are making a code contribution or large documentation contribution,
-please feel free to add yourself to the ``Misc/ACKS`` file alphabetically.
+`Lifecycle of a Pull Request <https://cpython-devguide.readthedocs.io/pullrequest/>`_.
```

Comment:
That's what happens when you copy from the wrong tab trying to update the URL. 😉